rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,150 @@
+
+### Sustainability Through the GA
+Upon general release, the full range of use cases will come into play and we anticipate a large increase in both the nubmer of tickets and diversity of questions asked by users. Addressing these tickets will require a big time committment, so we need an efficient system in place and a team well-versed in the DCP and its scientific applications. 
+
+This RFC describes an SOP that we think will satisfy both the Beta2 test phase and the early days of the GA. As we grow in our understanding of user support needs, this SOP will likely need revision to address long-term sustainability.
+
+### **User Stories**
+
+1.  > As a researcher new to the DCP, I would like to get a timely
+    > answer to my question so that I can have a successful experience
+    > submitting, exploring, downloading, or analyzing HCA data.
+
+2.  > As a DCP PM, PO or PL, I would like to be assured that Zendesk
+    > tickets are being responded to in a timely manner so that I can
+    > focus on my own tasks without having to micromanage responses.
+
+3.  > As a DCP team member, I would like to know whether or not I should
+    > be responding to a Zendesk ticket so that I can either respond in
+    > a timely manner or be assured that someone else is responding in a
+    > timely manner.
+
+## Scientific "guardrails" \[optional\]
+
+
+## **Detailed Design**
+
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
+
+## Issue Creation
+
+Issues in Zendesk originate from users of the DCP. These users could
+include data contributors, researchers trying to use the data in the
+DCP, software developers trying to leverage the DCP API. Any user of the
+DCP can file an issue. 
+
+While problems identified by an internal DCP staff member could be filed on Zendesk too, we favor a plan in which those issues are directly added to the applicable development
+repository, where they can be addressed directly. This plan eliminates the extra work for the triager of managing internally-generated tickets.
+
+Once an issue is
+created in Zendesk there is a tiered support network that kicks in to
+resolve the question.
+
+## Primary Support
+
+The triage person, or triage team, is responsible for receiving initial
+support requests from end users, and monitors the Zendesk system daily
+for new tickets. Note that the Zendesk system automatically sends out an initial canned response to the user, indicating that the request was received and is being reviewed by our support staff. 
+
+The Service Level Objective is to respond to the user with a solution no later than 48 business hours after it is filed.
+
+The triager takes ownership of each open ticket and does the following to develop a response:
+
+1. Evaluate whether there is enough information in the ticket to inform the
+response. If not, write back to the user and ask for the missing information. 
```

Comment:
How long do you wait for a response before closing the issue? Also - do you want to use Zendesk lingo - agent instead of triager?

Reply:
Added 2 weeks for the user to respond. I prefer to stay with terms that are not specific to Zendesk as we might change tools one day.